runtime/src/main/java/io/quarkus/jsch/runtime/JSchSessionRuntimeConfig.java

```diff
@@ -0,0 +1,57 @@
+package io.quarkus.jsch.runtime;
+
+import java.util.Map;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigDocMapKey;
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.smallrye.config.WithDefault;
+
+@ConfigGroup
+public interface JSchSessionRuntimeConfig {
+
+    /**
+     * The host to use for the JSch session.
+     */
+    @WithDefault("localhost")
+    public String host();
+
+    /**
+     * The port to use for the JSch session.
+     */
+    @WithDefault("22")
+    public int port();
+
+    /**
+     * The username to use for the JSch session.
+     */
+    public Optional<String> username();
+
+    /**
+     * The password to use for the JSch session.
+     */
+    public Optional<String> password();
+
+    /**
+     * The private key to use for the JSch session.
+     */
+    public Optional<String> key();
+
+    /**
+     * The passphrase to use for the JSch session.
+     */
+    public Optional<String> passphrase();
+
+    /**
+     * The configuration to use for the JSch session.
+     */
+    @ConfigDocMapKey("config-name")
+    public Map<String, String> config();
+
+    /**
+     * Mock the JSch session in dev and test mode.
+     */
+    @WithDefault("true")
+    public boolean mock();
```

Comment:
new doc:

```
/**
 * Mock the JSch session in dev and test mode:
 * This is useful for testing purposes with devservices where
 * we re-use the default session configuration generated by the devservice.
 */
```
Is it OK for you ?

Reply:
I understand the reason, but I guess it may be frustrating if you have set up 2 sessions and when you `quarkus dev` your app, you see that the second session is always connecting to the default session. Perhaps start a different SSH Server for each configured session?

Reply:
Also, start an SSH server only if the `host` is not defined (this is the same behavior as when defining the URL for Datasources)

Reply:
I think it is fixed